apps/files/src/views/FilesList.vue

```diff
@@ -256,7 +256,10 @@ export default Vue.extend({
 		},
 
 		dirContents(): Node[] {
-			return (this.currentFolder?._children || []).map(this.getNode).filter(file => file)
+			return (this.currentFolder?._children || [])
+				.map(this.getNode)
+				.filter(file => file)
+				.filter(({ attributes: { hidden } }) => hidden !== true)
```

Comment:
Sorry, fancy but unreadable.
```suggestion
				.filter(file => file?.attributes?.hidden !== true)
```